testing/tests/index.test.js

```diff
@@ -3,8 +3,14 @@ const path = require("path");
 
 const cheerio = require("cheerio");
 
+const { cli } = require("../../content");
```

Comment:
Perhaps this is so because it's a hack for now. You can add caporal to the `testing` workspace with `yarn workspace testing add caporal@1.4.0` and it'll share the same `/node_modules/caporal`.